sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/pubsublite/Publishers.java

```diff
@@ -42,7 +43,7 @@ private Publishers() {}
       checkArgument(token.isSupertypeOf(supplied.getClass()));
       return (Publisher<PublishMetadata>) supplied;
     }
-    return RoutingPublisherBuilder.newBuilder()
+    return new PartitionCountWatchingPublisher(PartitionCountWatchingPublisherSettings.newBuilder()
```

Comment:
With this change, is there anywhere else using `RoutingPublisher`?

Reply:
I don't think so. I thought maybe the java client library used it if the user configured a static set of partitions, but it doesns't seem to do so